docs/details/blocks/Math/Add.md

```diff
@@ -0,0 +1,40 @@
+Addition is a binary operation, i.e. an operation that requires exactly two arguments.
+
+In this case `Math.Add` is the binary operator, the 1st argument is `:Input`, and the 2nd argument is the parameter `:Operand`. The `:Output` is the result of this operation over these two arguments and generally is of the same type as the `:Input`. Each argument to this operator can be an integer, a float, or a sequence of such entities (type of both `:Input` and `:Operand` must match).
```

Comment:
This is quite confusing since there aren't anything like `:Input` and `:Output`. Those are not parameters to the blocks.

Blocks do have an input value and an output value but they are not named.

Reply:
Yes, agreed. I misnamed them. I have corrected this and fixup'ed it in the last commit.